node/pkg/guardiansigner/amazonkms.go

```diff
@@ -118,6 +118,12 @@ func NewAmazonKmsSigner(ctx context.Context, unsafeDevMode bool, keyPath string)
 		return nil, fmt.Errorf("Failed to unmarshal KMS public key: %w", err)
 	}
 
+	// It is possible to use `ethcrypto.UnmarshalPubkey(asn1Pubkey.PublicKey.Bytes)`` to get the public key,
+	// but `UnmarshalPubkey()` uses elliptic.Unmarshal() internally, which has been marked as deprecated.
+	// The following code implements similar logic, with the indexes meaning the following:
+	// 0: The first byte is the prefix byte, which is 0x04 for uncompressed keys.
+	// 1-32: The next 32 bytes are the X coordinate.
+	// 33-64: The next 32 bytes are the Y coordinate.
```

Comment:
Thanks for the extra docstring. Is it worth doing an explicit length check on the byte array before slicing from it? Or is the length guaranteed in package code?

Reply:
I wouldn't feel comfortable making that assumption after checking the code. Added a minimum length check.